index.bs

```diff
@@ -4231,6 +4231,7 @@ promises returned by its <a lt="writable stream writer">writer</a>'s {{WritableS
     return new WritableStream({
       start(controller) {
         ws.onerror = () => controller.error(new Error("The WebSocket errored!"));
+        ws.onclose = () => controller.error(new Error("The server closed the connection unexpectedly!"));
```

Comment:
If for example the remote server closes the connection abruptly, then onerror and onclose will both fire.

I don't actually know of a case where onerror fires without onclose also firing. Certainly Chrome's implementation never does it. So one way of simplifying this would just be to not set an onerror handler.